apps/shipments/templates/email/shipment_link.html

```diff
@@ -374,7 +374,7 @@
                                     <tr>
                                       <td style="width:40px;">
                                         <a href="https://twitter.com/shipchain" target="_blank">
-                                          <img height="auto" src="{% fullstatic 'icon/twitter_orange.png' %}" style="border:0;display:block;outline:none;text-decoration:none;height:auto;width:100%;font-size:13px;" width="40" />
+                                          <img height="auto" src="{% fullstatic 'icon/tweet_orange.png' %}" style="border:0;display:block;outline:none;text-decoration:none;height:auto;width:100%;font-size:13px;" width="40" />
```

Comment:
I would suggest changing it to "ShipChain_twitter_orange", or maybe "twitter_icon_orange". Would have more clarity.